test_bin/data/n3h_config.json

```diff
@@ -1,12 +1,14 @@
 {
   "backend_kind": "N3H",
   "backend_config": {
-    "socketType": "ws",
-    "spawn": {
-      "env": {
-        "N3H_MODE": "REAL",
-        "N3H_IPC_SOCKET": "tcp://127.0.0.1:*",
-        "N3H_LOG_LEVEL": "t"
+    "Json": {
```

Comment:
This got introduced when the lib3h config option got added, and test_bin hasn't been able to run against n3h without this fix since. Conductor was updated properly already.